lib/cocoapods/validator.rb

```diff
@@ -1048,7 +1050,7 @@ def parse_xcodebuild_output(output)
     # @return [String] Executes xcodebuild in the current working directory and
     #         returns its output (both STDOUT and STDERR).
     #
-    def xcodebuild(action, scheme, configuration)
+    def xcodebuild(action, scheme, configuration, deployment_target)
```

Comment:
lol love how this is the only change to the method. variable shadowing ftw?

Reply:
nit: use a named parameter ?

Reply:
love it